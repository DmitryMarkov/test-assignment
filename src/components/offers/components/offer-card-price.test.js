import React from 'react'
import { render } from '@testing-library/react'
import OfferCardPrice from './offer-card-price'

test('OfferCardPrice should render properly', () => {
  const mockNights = 7
  const mockPrice = 4200
  const { asFragment, getByText } = render(
    <OfferCardPrice nights={mockNights} price={mockPrice} />
  )

  expect(getByText(`${mockNights} nights`)).toBeInTheDocument()
  expect(asFragment()).toMatchSnapshot()
})
