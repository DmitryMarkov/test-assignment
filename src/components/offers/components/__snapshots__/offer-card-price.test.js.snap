// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferCardPrice should render properly 1`] = `
<DocumentFragment>
  <div
    class="css-tsqj0x"
  >
    <div
      class="css-bbnvqk"
    >
      7 nights
    </div>
    <div
      class="css-fm3b53"
    >
      â‚¬ 4,200
    </div>
  </div>
</DocumentFragment>
`;
